tests/large/test_ide.py

```diff
@@ -326,6 +326,25 @@ def setUp(self):
         self.name = 'DataGrip'
 
 
+class RiderIDETests(IdeaIDETests):
+    """Rider from the IDE collection."""
+
+    TIMEOUT_INSTALL_PROGRESS = 120
+    TIMEOUT_START = 60
+    TIMEOUT_STOP = 60
+
+    def setUp(self):
+        super().setUp()
+        self.installed_path = os.path.join(self.install_base_path, "ide", "rider")
+        self.desktop_filename = 'jetbrains-rider.desktop'
+        self.command_args = '{} ide rider'.format(UMAKE)
+        self.name = 'Rider'
+
+    def test_default_install(self):
```

Comment:
This can now be removed, so we don't override the default.
In the gogland PR I added a "check if there is a stable version" in the installer